### PR TITLE
to do  ticket list Project &&  update Project  && create Project&& delete 

### PR DESCRIPTION
**Pull request by lethu96**
to do ticket Update project

### CODE REVIEW

File: resources/assets/js/components/TableRowMember.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        super(props);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleSubmit(event)
+    {
+        event.preventDefault();
+        let uri = MyGlobleSetting.url + '/api/member/${this.props.obj.id}';
+        axios.delete(uri);
+         browserHistory.push('/display-item-member');
```
Comment (jackdiff) [inline resources/assets/js/components/TableRowMember.js]: Wrong indent

File: resources/assets/js/components/UpdateProject.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+import { Link } from 'react-router';
+
+
+class UpdateProject extends Component
+{
+    constructor(props)
+    {
+        super(props);
+        this.state = {name: '', information: '', deadline: '', type: '', status: ''};
+        this.handleChangeName = this.handleChangeName.bind(this);
```
Comment (jackdiff) [inline resources/assets/js/components/UpdateProject.js]: Why do you using bind(this) ?

File: app/Http/Controllers/ProjectController.php [hunk 3]
```diff
@@ -44,32 +44,22 @@ public function store(StoreCreateProject $request)
...
+            $project->type = $data['type'];
+            $project->status = $data['status'];
+            $project->save();
+             return response()->json('Project Updated Successfully.');
     }
+
     public function edit($id)
     {
         $project = Project::find($id);
-        return response()->json($project);
```
Comment (jackdiff) [inline app/Http/Controllers/ProjectController.php]: Why you don not return a json response ?

**Review by jackdiff — CHANGES_REQUESTED** [review]

File: resources/assets/js/components/UpdateProject.js (hunk 2 above)
Comment (lethu96) [reply resources/assets/js/components/UpdateProject.js]: i will use it for  validate input

File: resources/assets/js/components/TableRowMember.js (hunk 1 above)
Comment (lethu96) [reply resources/assets/js/components/TableRowMember.js]: yes, i have fix it

File: app/Http/Controllers/ProjectController.php (hunk 3 above)
Comment (lethu96) [reply app/Http/Controllers/ProjectController.php]: i have  return json

File: resources/assets/js/components/UpdateProject.js (hunk 2 above)
Comment (jackdiff) [reply resources/assets/js/components/UpdateProject.js]: Do you try to remove it and run ?

File: app/Http/Controllers/ProjectController.php (hunk 3 above)
Comment (jackdiff) [reply app/Http/Controllers/ProjectController.php]: You're return an Eloquent object ?

**Review by jackdiff — CHANGES_REQUESTED** [review]

File: resources/assets/js/components/UpdateProject.js (hunk 2 above)
Comment (lethu96) [reply resources/assets/js/components/UpdateProject.js]: yes

Comment (lethu96) [reply resources/assets/js/components/UpdateProject.js]: when i remote , it not run

File: resources/assets/js/components/UpdateProject.js [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+        return (
+            <div>
+                <h1>Update Project</h1>
+                <div className="row">
+                    <div className="col-md-10"></div>
+                    <div className="col-md-2">
+                        <Link to="/display-item" className="btn btn-success">Return to Project</Link>
+                    </div>
+                </div>
+                <form onSubmit={this.handleSubmit} name="_method" value="PUT">
```
Comment (jackdiff) [inline resources/assets/js/components/UpdateProject.js]: No need name="_method" and value= "PUT"

**Review by jackdiff — CHANGES_REQUESTED** [review]

**Review by jackdiff — APPROVED** [review]

File: resources/assets/js/components/DisplayProject.js [hunk 5]
```diff
@@ -15,17 +15,18 @@ class DisplayProject extends Component
     {
         axios.get('http://localhost:8000/project')
         .then(response => {
-            this.setState({ project: response.data });})
+                this.setState({ project: response.data });
+            })
```
Comment (hieutq) [inline resources/assets/js/components/DisplayProject.js]: indentation wrong. Pls fix this

**Review by hieutq — APPROVED** [review]

File: resources/assets/js/components/CreateProject.js [hunk 6]
```diff
@@ -27,6 +29,7 @@ class CreateProject extends Component
         this.setState({
             name: e.target.value
         })
+        console.log(e.target.value)
```
Comment (hieutq) [inline resources/assets/js/components/CreateProject.js]: Pls remove this line.

**Pull request merged**